server/controllers/apply/applicationsController.ts

```diff
@@ -83,4 +84,24 @@ export default class ApplicationsController {
       })
     }
   }
+
+  submit(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      const application = await this.applicationService.findApplication(req.user.token, req.params.id)
+      application.document = getResponses(application)
+
+      // TODO: validate that the user has confirmed information is complete
```

Comment:
I'm not a big fan of leaving TODO's in but in this case we definitely need to come back to this 🙈